docs/reference/announcements/860.md

```diff
@@ -10,6 +10,10 @@ Important changes and updates for the Camunda 8.6 release are summarized below.
 | :------------- | :----------------- | :--------------------------------------------------- |
 | 8 October 2024 | 14 April 2026      | [8.6 release notes](/reference/release-notes/860.md) |
 
+## Orchestration Cluster Components must run on exact same minor and patch levels
+
+With version `8.6.0` going forward, the core Orchestration Cluster components (`Zeebe`, `Operate`, `Tasklist`) must run the exact same `minor` and `patch` level to ensure compatability. Please see [component version matrix](../supported-environments.md#component-version-matrix).
+
```

Comment:
We don't define an [orchestration cluster](https://docs.camunda.io/docs/8.7/reference/glossary/#orchestration-cluster) or [orchestration core](https://docs.camunda.io/docs/8.7/reference/glossary/#orchestration-core) until 8.7+, so we should just list the components.